client/src/components/themeSwitcher/ThemeSwitcher.jsx

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
+import classes from './themeSwitcher.module.css';
+import darkModeBtn from '../../../public/images/darkModeBtn.png';
+import lightModeBtn from '../../../public/images/lightModeBtn.png';
 
-function ThemeSwitcher() {
-  return <div></div>;
+function ThemeSwitcher({ theme, themeToggle }) {
+
+  const themeBtnImg = theme == 'light' ? darkModeBtn : lightModeBtn;
+
+  return (
+    <div className={classes.container}>
+      <div className={classes.themeBtn} onClick={themeToggle}>
+        <img src={themeBtnImg}></img>
```

Comment:
IMG tag doesn't use a closing tag. Validagor may to give you warning.

Reply:
Though tonce again it is not and html img!

Reply:
changed to <img/>